src/Microsoft.Restier.Core/ApiBuilderExtensions.cs

```diff
@@ -152,11 +152,7 @@ public static ApiBuilder AddContributor<T>(this ApiBuilder obj, ApiServiceContri
             Ensure.NotNull(obj, "obj");
             Ensure.NotNull(contributor, "contributor");
 
-            // Services have singleton lifetime by default, call Make... to change.
-            obj.Services.TryAddSingleton(typeof(T), ChainedService<T>.DefaultFactory);
-            obj.Services.AddInstance(contributor);
-
-            return obj;
+            return obj.AddContributorNoCheck<T>(contributor);
```

Comment:
Please remove the unused using in this file: `using System.Collections.Generic;`